### PR TITLE
bugfixes

### PR DESCRIPTION
**Pull request by cardanians**

### CODE REVIEW

File: doc/stake-pool-operations/withdraw-rewards.md [hunk 1]
```diff
@@ -27,17 +27,6 @@ You'll withdraw rewards into a payment.addr wich will pay for the transaction fe
...
-### Draft the withdraw transaction to transfer the rewards to a payment.addr
-
-    cardano-cli shelley transaction build raw \
-    --tx-in a82f8d2a85cde39118a894306ad7a85ba40af221406064a56bdd9b3c61153527#1
-    --tx-out $(cat payment.addr)+0
-    --withdrawal $(cat stake.addr)+550000000
-    --ttl 0
-    --fee 0
-    --out-file withdraw_rewards.raw
-
```
Comment (intricate) [inline doc/stake-pool-operations/withdraw-rewards.md:40]: Isn't this required for calculating the transaction fee?

File: doc/stake-pool-operations/withdraw-rewards.md [hunk 2]
```diff
@@ -53,11 +42,11 @@ You'll withdraw rewards into a payment.addr wich will pay for the transaction fe
...
 ### Build the raw transaction.
 
-    expr 194054070 - 171089
-    193882981
+    expr 194054070 - 171089 + 550000000
+    743882981
 
     cardano-cli shelley transaction build raw \
-    --tx-in $(cat payment.addr)#1
+    --tx-in a82f8d2a85cde39118a894306ad7a85ba40af221406064a56bdd9b3c61153527#1
```
Comment (intricate) [inline doc/stake-pool-operations/withdraw-rewards.md:49]: :+1: this is definitely a mistake.

**Review by intricate — CHANGES_REQUESTED** [review]
Could you perhaps amend your commit message in order to describe the changes?

File: doc/stake-pool-operations/withdraw-rewards.md (hunk 1 above)
Comment (cardanians) [reply doc/stake-pool-operations/withdraw-rewards.md]: true true.  thanks.

**Review by intricate — APPROVED** [review]

**Review by intricate — CHANGES_REQUESTED** [review]
Apologies, didn't mean to approve just yet. Let's squash these commits and amend the message.

**Comment by intricate** [comment]
Superseded by https://github.com/input-output-hk/cardano-node/pull/1962.

@cardanians: You've been included as a co-author on that new PR.

**Pull request closed without merging**